encrypt/.gitignore

```diff
@@ -0,0 +1 @@
+/build/
```

Comment:
We allready have .gitignore file with excluded build folder.
https://github.com/coders-school/password-manager/blob/main/.gitignore